### ISSUE TITLE
Add a slider, and allow configuration for no mousewheel interaction

### ISSUE THREAD

**Issue by Aiue** [Aiue/AQT#33]
And the slider should only be shown if part of the scrollchild is hidden, obviously.

**Comment by dalesnail** [comment]
Just to add to the feature here, would be nice to also have an option to increase the scroll speed on the tracker. Love the scrolling, but it is quite slow to scroll.

**Comment by Aiue** [comment]
Yeah, I've been meaning to add configurability for it, I just keep forgetting. :sweat_smile:
